tests/sqllogictests/suites/base/20+_others/20_0017_describe_view.test

```diff
@@ -0,0 +1,156 @@
+statement ok
```

Comment:
Good test 👍 
It would be better to move this test into `tests/sqllogictests/suites/base/05_ddl/05_0022_ddl_desc_view.test`?

Reply:
but `desc_table.test`  in the same package which named 20+_others.

Reply:
`desc_table.test` is not a good example, will migrate it soon.

Reply:
ok, i will move it